app/controllers/games_controller.rb

```diff
@@ -10,7 +10,8 @@ def new
   end
 
   def create
-    @game = Game.create(game_params.merge(user_id: current_user.id))
+    @opponent = User.create(email: 'opponent@example.com', password: 'password')
```

Comment:
If I'm reading this correctly, does this populate a dummy opponent account so all the pieces will populate once the game is created? Then update with whichever `current_user` joins a game later? I've been pondering how to get around that limitation of `populate_board!` and like this solution a lot.

Reply:
Yeah that's kind of what I had in mind. I wasn't thinking too far ahead on how to actually add the opponent but this is necessary just to get the pieces on the board. It can be modified later.

Reply:
That makes sense and I think it's fine for now. Perhaps there is a way to modify the `populate_board!` game model method, but I agree that can come later if we have time.